lib/constants.go

```diff
@@ -123,6 +123,11 @@ var (
 	// transfer txns, NFT burn txns, and AuthorizeDerivedKey txns will be accepted.
 	// Triggers: 12PM PT on 9/15/2021
 	NFTTransferOrBurnAndDerivedKeysBlockHeight = uint32(60743)
+
+	// BuyNowNFTBlockHeight defines the height at which NFTs can be sold at a fixed price instead of an auction style.
+	// Note: this will be adjusted, 90000 is a placeholder until a block height is decided.
+	// Triggers: TBD on TBD
+	BuyNowNFTBlockHeight = uint32(90000)
```

Comment:
Just to make it foolproof, I'd put a really high value here for now with a FIXME:

```
	// BuyNowNFTBlockHeight defines the height at which NFTs can be sold at a fixed price instead of an auction style.
	// FIXME: Currently set to a really high value until we decide when we want this to trigger.
	BuyNowNFTBlockHeight = math.MaxUint32
```